Add diagram. (#14615)
cockroachdb/docs: v22.1/refresh.md

The user must be the [owner](owner-to.html) of the materialized view or have [admin](security-reference/authorization.html#admin-role) privileges.

## Synopsis

<div>
{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/refresh_materialized_views.html %}

> </div>

Reviewer: Is there a reason this needs div tags?

Author: I just copied it from another page.

## Parameters

 Parameter | Description
-----------|-------------

> `opt_concurrently` | `CONCURRENTLY` (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.

Reviewer: I know that "no-op" is a commonly used phrase, but do you think it bears spelling out the meaning for a general reader?

Reviewer: Also, should we not italicize "Default behavior" per our style usage?

> `opt_concurrently` | `CONCURRENTLY` (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.

Author: ```suggestion
`opt_concurrently` | `CONCURRENTLY` (Default behavior) This keyword has no effect. It is present for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
```

`view_name` | The name of the materialized view to refresh.

> `opt_clear_data` | `WITH DATA` (*Default behavior*) Refresh the stored query results. <br>`WITH NO DATA` Drop the query results of the materialized view from storage.

Author: ```suggestion
`opt_clear_data` | `WITH DATA` (Default behavior) Refresh the stored query results. <br>`WITH NO DATA` Drop the query results of the materialized view from storage.
```
